src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -52,7 +51,6 @@ private void CleanUpPersistentFolder()
         {
         }
 
-        [Fact]
```

Comment:
I'm apparently an idiot.  I think i moved them to the abstract class, and *removed* the fact attribute, and then (obviously) couldn't have them run from the derived type.   Ugh.  sorry about that.